packages/fbjs/src/__forks__/setImmediate.js

```diff
@@ -14,4 +14,6 @@
 // setimmediate adds setImmediate to the global. We want to make sure we export
 // the actual function.
 require('setimmediate')
-module.exports = global.setImmediate;
+module.exports = function(fn, ms) {
+  global.setImmediate(fn, ms);
+};
```

Comment:
This is fine without arrow functions but we're losing the ability to pass rest args. Let's use rest and call spread to keep them. This should work:

```js
module.exports = (...args) => global.setImmediate(...args);
```